src/adapters/MeetingsJSONAdapter.js

```diff
@@ -255,6 +269,38 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     }
   }
 
+  /**
+   * Joins the meeting without camera
+   *
+   * @param {string} ID ID of the meeting for which to join
+   * @private
+   */
+  async joinMeetingWithoutCamera(ID) {
+    if (this.datasource[ID]) {
+      const meeting = this.datasource[ID];
+
+      meeting.state = MeetingState.JOINED;
+
+      document.dispatchEvent(new CustomEvent(JOIN_WITHOUT_CAMERA_CONTROL, {detail: meeting}));
```

Comment:
Does something listen to this event? What is supposed to happen when the event is dispatched?

Reply:
I don't think there's anything specifically responding to such events, but neither is for the normal `JOIN_CONTROL`  events. The meeting is joined here and an event is triggered so that they are all coalesced and an updated meeting object is emitted on line 239.

https://github.com/webex/components/pull/232/files#diff-b63aa3e7e2482c57f9d72c2a873dafae6bdaedaa8c04639c9313b922df39ebd9R217